### PR TITLE
Video chat is happening

### PR DESCRIPTION
**Pull request by haleyjung**

### CODE REVIEW

**Comment by katyfsy** [comment]
🥳🥳🥳🥳we're ready to party!

**Review by katyfsy — APPROVED** [review]
lgtm

File: client/pages/BookClubDetail/index.jsx [hunk 1]
```diff
@@ -7,101 +7,104 @@ import ForumIcon from '@mui/icons-material/Forum';
...
-
-  }
+      });
+    setMyClub(bookclubDetails);
+    setRoomName(bookclubDetails.bookclubInfo.bookclubName);
+  };
 
   const getBookclubs = (uid) => {
     axios
       .get('http://localhost:3030//myBookclubs', { params: { userId: 'qwew' } })
```
Comment (addisonhernandez) [inline client/pages/BookClubDetail/index.jsx:42]: does this work with the double `//`?

File: client/pages/Chat/LiveChat.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import io from "socket.io-client";
+import Chat from "./ChatRoom";
+
+const socket = io.connect('http://localhost:3030');
+
+export default function LiveChat({ user, roomName }) {
```
Comment (addisonhernandez) [inline client/pages/Chat/LiveChat.jsx:7]: :speech_balloon: !!!

**Review by addisonhernandez — APPROVED** [review]

**Comment by addisonhernandez** [comment]
[Implement video chat functionality](https://trello.com/c/osn9Czpi/41-implement-video-chat-functionality)

**Pull request merged**